docs/develop/specify-office-hosts-and-api-requirements.md

```diff
@@ -128,24 +128,24 @@ The following code example shows an add-in that loads in all Office host applica
 
 - The  **Method** element specifies an individual method that must be supported in the Office host where your add-in runs. The **Name** attribute is required and specifies the name of the method qualified with its parent object.
 
-
 ## Use runtime checks in your JavaScript code
 
-
 You might want to provide additional functionality in your add-in if certain requirement sets are supported by the Office host. For example, you might want to use the Word JavaScript APIs in your existing add-in if your add-in runs in Word 2016. To do this, you use the [isSetSupported](/javascript/api/office/office.requirementsetsupport#issetsupported-name--minversion-) method with the name of the requirement set. **isSetSupported** determines, at runtime, whether the Office host running the add-in supports the requirement set. If the requirement set is supported, **isSetSupported** returns **true** and runs the additional code that uses the API members from that requirement set. If the Office host doesn't support the requirement set, **isSetSupported** returns **false** and the additional code won't run. The following code shows the syntax to use with **isSetSupported**.
 
-
 ```js
-if (Office.context.requirements.isSetSupported(RequirementSetName, VersionNumber))
+if (Office.context.requirements.isSetSupported(RequirementSetName, MinimumVersion))
 {
    // Code that uses API members from RequirementSetName.
 }
 
 ```
 
--  _RequirementSetName_ (required) is a string that represents the name of the requirement set. For more information about available requirement sets, see [Office Add-in requirement sets](/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets).
-    
--  _VersionNumber_ (optional) is the version of the requirement set.
+- _RequirementSetName_ (required) is a string that represents the name of the requirement set. For more information about available requirement sets, see [Office Add-in requirement sets](/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets).
+- _MinimumVersion_ (optional) is the version of the requirement set.
```

Comment:
Perhaps consider specifying 'string' here as the data type for this parameter (just to reinforce the message that it should be a string) and revising the description to be a bit more specific?  

_MinimumVersion_ (optional) a string that specifies the requirement set version that the host must support in order for the code within the `if` statement to run (e.g., **"1.9"**).